vpc.tf

```diff
@@ -120,7 +120,7 @@ resource "aws_security_group_rule" "sg_ingress_public_443" {
   from_port         = 443
   to_port           = 443
   protocol          = "tcp"
-  cidr_blocks       = ["0.0.0.0/0"]
+  cidr_blocks       = ["<cidr>"]
```

Comment:
**Tenable.cs remediation**
 
**[LOW]: Ensure Security Groups do not have unrestricted specific ports open - (HTTPS,443)**
 
**Vulnerability**
Security Groups - Unrestricted Specific Ports - HTTPS,443. Exposed ports may allow attackers to access the application listening on that port.
 
**Remediation**
In AWS Console - 
 1. Sign in the AWS Console and go to the Amazon VPC console.
 2. In the navigation pane, select Security Groups.
 3. For each security group, perform the following actions:
      a) Select the security group.
      b) Select the Inbound Rules.
      c) Identify the rules to be removed.
      d) Remove the required rules.
 4. Click Save.

source: tenablecs-